### PR TITLE
feat: IncludesCI Script 

### LINKED ISSUE #14: Add a CI Tool ex: CircleCI, TravisCI
**What is CI?**
Continuous integration (CI) is the practice of automating the integration of code changes from multiple contributors into a single software project.

### PR DESCRIPTION
**Pull request by s-ayush2903**
**Fixes #14** 
**Changes:** A simple config.yaml file was written as CI script

**Screenshots for the change:** N/A

### CODE REVIEW

**Review by ShivangiSingh17 — CHANGES_REQUESTED** [review]
Add Travis CI also as per mentioned in the #14

**Review by ShivangiSingh17 — COMMENTED** [review]
make the file name to be .travis.yml

File: travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    - tools
+
+    - build-tools-27.1.1
+    - android-27
+
+    # Support library
+    - extra-android-support
+    - extra-android-m2repository
+script:
+  - chmod +x ./gradlew
```
Comment (ShivangiSingh17) [inline travis.yml]: Please parse yml files through [YAML Formatter,](https://jsonformatter.org/yaml-formatter) the merged .travis.yml had basic syntax issues. Please make sure it passes [YAML Lint ](http://www.yamllint.com/)before committing further changes.

YML is a very unforgiving language, it looks deeply into indentation and stuff just like Python. Be sure, and careful :)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    - tools
+
+    - build-tools-27.1.1
+    - android-27
+
+    # Support library
+    - extra-android-support
+    - extra-android-m2repository
+script:
+  - chmod +x ./gradlew
```
Comment (ShivangiSingh17) [inline .travis.yml]: Please parse yml files through YAML Formatter, the merged .travis.yml had basic syntax issues. Please make sure it passes YAML Lint before committing further changes.

YML is a very unforgiving language, it looks deeply into indentation and stuff just like Python. Be sure, and careful :)

**Comment by s-ayush2903** [comment]
@somenath1435 please review

**Review by ShivangiSingh17 — APPROVED** [review]

**Pull request merged**